drivers/hmis/app/models/hmis/hud/processors/client_processor.rb

```diff
@@ -49,6 +49,9 @@ def process(field, value)
       when 'mci_id'
         process_mci(value)
         {}
+      when 'veteran_status'
```

Comment:
Here's the comment in the HMIS CSV spec:
```
Export 99 (Data not collected) for all clients, including minors, for whom there is no Veteran Status data.
```
So I think this is the correct approach

Reply:
Yes, it just means we will need to have special cases in every processor where this occurs. Hopefully its not very common, I think veteran status is an outlier.

Reply:
We could update the db to default to 99

Reply:
oh I like that approach much better!

Reply:
although that alone is not enough because we run the validation before saving to the db. so we'd need AR to set the default attribute somehow. tried with https://github.com/greenriver/hmis-warehouse/compare/release-86...gig/default-vet-status-to-99 but I"m going to merge this for now. I think the db default makes sense though, in general. DisablingCondition is the only field that has the same special note in the spec, so I included it.